components/must-read/must-read.less

```diff
@@ -0,0 +1,120 @@
+@import (reference) '../../styles/variables';
+
+.must-read {
+	max-width: 450px;
+	margin: @spacing-large 0;
+	width: 100%;
+	padding: @spacing-default;
+
+	@media @screen-medium{
+		width: calc(~"(50% - 15px)");
+		vertical-align: top;
+		margin-left: @spacing-medium;
+		margin-top: 0;
+		padding: 0 @spacing-medium;
+	}
+
+	@media @screen-medium-large {
+		width: calc(~"((100%/3) - 30px)");
```

Comment:
Maybe turn 3 into a var so it's more readable because I don't know what that 3 stands for